ts/webui/src/components/managementExp/Experiment.tsx

```diff
@@ -0,0 +1,433 @@
+                <Hearder />
+                {errorMessage !== undefined ? (
+                    <div className='warning'>
+                        <MessageInfo info={errorMessage} typeInfo='error' />
+                    </div>
+                ) : null}
+                <Stack className='contentBox expBackground'>
+                    <Stack className='content'>
+                        <Stack className='experimentList'>
+                            <Stack className='box' horizontal>
+                                <div className='search'>
+                                    <SearchBox
+                                        className='search-input'
+                                        placeholder='Search the experiment by name and ID'
+                                        onEscape={this.setOriginSource.bind(this)}
+                                        onClear={this.setOriginSource.bind(this)}
+                                        onChange={this.searchNameAndId.bind(this)}
+                                    />
+                                </div>
+                                <div className='filter'>
+                                    <DefaultButton
+                                        onClick={this.clickFilter.bind(this)}
+                                        className={`${!hideFilter ? 'filter-button-open' : null}`}
+                                    >
+                                        <Icon iconName='Equalizer' />
+                                        <span className='margin'>Filter</span>
+                                    </DefaultButton>
+                                </div>
+                            </Stack>
+                            <Stack className={`${hideFilter ? 'hidden' : ''} filter-condition`} horizontal gap={25}>
+                                <FilterBtns
+                                    platform={platform}
+                                    selectedStatus={selectedStatus}
+                                    selectedPlatform={selectedPlatform}
+                                    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                                    selectedStartDate={selectedStartDate!}
+                                    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                                    selectedEndDate={selectedEndDate!}
+                                    selectStatus={this.selectStatus.bind(this)}
+                                    selectPlatform={this.selectPlatform.bind(this)}
+                                    getSelectedData={this.getSelectedData.bind(this)}
+                                    setSearchSource={this.setSearchSource.bind(this)}
+                                />
+                            </Stack>
+                            <DetailsList
+                                columns={this.columns}
+                                items={source}
```

Comment:
Is this supposed to be auto-refreshing?

Reply:
in this version, it has not auto-refresh